flink-kubernetes/src/test/java/org/apache/flink/kubernetes/kubeclient/factory/KubernetesJobManagerFactoryTest.java

```diff
@@ -468,4 +468,37 @@ void testSetJobManagerDeploymentReplicas() throws Exception {
         assertThat(kubernetesJobManagerSpecification.getDeployment().getSpec().getReplicas())
                 .isEqualTo(JOBMANAGER_REPLICAS);
     }
+
+    @Test
```

Comment:
The new added two tests might be more appropriate to be put in `InitJobManagerDecoratorTest`.